src/components/pages/filesytem/FileIcon.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {ReactElement} from "react";
+import {
+    FileEarmarkCodeIcon,
+    FileEarmarkIcon,
+    FileEarmarkImageIcon,
+    FileEarmarkLock2Icon,
+    FileEarmarkMusicIcon,
+    FileEarmarkPDFIcon,
+    FileEarmarkPlayIcon,
+    FileEarmarkRichtextIcon,
+    FileEarmarkTextIcon,
+    FileEarmarkZipIcon,
+    FolderIcon
+} from "../../../assets/images/icons/reactSvgIcons/SymbolFile";
+import {reverseString} from "../../../background/methods/strings";
+
+const ICON_PREFERENCES = {
+    height: "40px",
+    width: "100%",
+    color: "secondary"
+};
+
+function FileIcon(isFolder: boolean, name: string): ReactElement {
```

Comment:
Nice that this is a component now, the backend also provides information about the exact mimetype and the general type, we could use this to improve this (then you dont have to list all the file extensions )